components/datadog/fakeintake/ecsFargate.go

```diff
@@ -14,41 +16,101 @@ const (
 	port          = 80
 )
 
-const (
-	DefaultFakeintakePort = 80
-)
+type Component struct {
+	pulumi.ResourceState
 
-func FargateLinuxTaskDefinition(e aws.Environment, name string) (*ecs.FargateTaskDefinition, error) {
-	return ecs.NewFargateTaskDefinition(e.Ctx, e.Namer.ResourceName(name), &ecs.FargateTaskDefinitionArgs{
-		Containers: map[string]ecs.TaskDefinitionContainerDefinitionArgs{
-			containerName: *fargateLinuxContainerDefinition(),
-		},
-		Cpu:    pulumi.StringPtr("256"),
-		Memory: pulumi.StringPtr("512"),
-		ExecutionRole: &awsx.DefaultRoleWithPolicyArgs{
-			RoleArn: pulumi.StringPtr(e.ECSTaskExecutionRole()),
+	Host pulumi.StringOutput
+}
+
+func FargateLinuxComponentDefinition(e aws.Environment) (*Component, error) {
```

Comment:
```suggestion
func NewECSFargateInstance(e aws.Environment) (*Component, error) {
```

Reply:
That’s definitively a better name: https://github.com/DataDog/test-infra-definitions/pull/225/commits/e0a571d3c744c8b18fa3d40094eef2be83091f14.